preliz/internal/plot_helper.py

```diff
@@ -606,8 +606,8 @@ def check_inside_notebook(need_widget=False):
             )
         if need_widget:
             shell_name = shell.__class__.__name__
-            if shell_name == "ZMQInteractiveShell" and "nbagg" not in get_backend():
-                msg = f"To run {name}, you need use the magic `%matplotlib widget`"
+            if shell_name == "ZMQInteractiveShell" and get_backend() not in ["ipympl", "widget"]:
```

Comment:
I think the backend should be `widget` or `ipympl` for interactive mode.

Reply:
If this fixing a concrete issue? I don't remember having problems with this

Reply:
Strange, `%matplotlib widget` sets the backend to `widget`
So it will raise a runtime error

Reply:
When will this be an issue? If I do something like 

```python
%matplotlib widget
result = pz.Roulette()
```python

It will work without issues.

Reply:
Yes, it will show the plot in interactive mode...
But it prompts a Runtime Error Message

Reply:
This RuntimeError popped up when I run it

![image](https://github.com/user-attachments/assets/c2bff55e-55d4-42e0-9a5f-df95487b4ad7)

Reply:
I don't see any error, and the plots work just fine.

Reply:
Should I add a check for `nbagg`, `widget` and `ipympl`. The documentation is giving the above error when it is set to only `nbagg`. Also `%matplotlib ipympl` is also supported for interactive mode. 
This should cover all our cases..

What do you think?

Reply:
With the changes you are proposing I get a runtime error

Reply:
Hmm...
I just rebuilt the [documentation](https://preliz--627.org.readthedocs.build/en/627/examples/gallery/ppa.html), I am getting this.

Reply:
maybe a version issue? yes, that may work

Reply:
Could be, have made the change
This is now resolved